test/Common.FunctionalTests/ConfigurationChangeTests.cs

```diff
@@ -71,8 +77,48 @@ public async Task OutOfProcessToInProcessHostingModelSwitchWorks()
                 .SetAttributeValue("hostingModel", "inprocess"));
 
             // Have to retry here to allow ANCM to receive notification and react to it
-            // Verify that worker process gets restarted with new process id
+            // Verify that inprocess application was created and tried to start
             await deploymentResult.HttpClient.RetryRequestAsync("/HelloWorld", r => r.StatusCode == HttpStatusCode.InternalServerError);
+
+            StopServer();
+            EventLogHelpers.VerifyEventLogEvent(deploymentResult, TestSink, "Could not find the assembly 'aspnetcorev2_inprocess.dll'");
+        }
+
+        [ConditionalTheory]
+        [InlineData(HostingModel.InProcess)]
+        [InlineData(HostingModel.OutOfProcess)]
+        public async Task ConfigurationTouchedStress(HostingModel hostingModel)
+        {
+            var deploymentResult = await DeployAsync(_fixture.GetBaseDeploymentParameters(hostingModel, publish: true));
+
+            await deploymentResult.AssertStarts();
+            var load = Helpers.StressLoad(deploymentResult.HttpClient, "/HelloWorld", response => {
+                var statusCode = (int)response.StatusCode;
+                Assert.True(statusCode == 200 || statusCode == 503, "Status code was " + statusCode);
+            });
+
+            for (int i = 0; i < 100; i++)
+            {
+                // AddAppOffline might fail if app_offline is being read by ANCM and deleted at the same time
```

Comment:
I mentioned this before, but having an equivalent check with apphost.config would be useful.